content/docs/user-guide/external-dependencies.md

```diff
@@ -149,12 +149,12 @@ $ dvc import-url https://data.dvc.org/get-started/data.xml
 Importing 'https://data.dvc.org/get-started/data.xml' -> 'data.xml'
 ```
 
-The command above creates the <abbr>import stage</abbr> (DVC-file)
-`data.xml.dvc`, that uses an external dependency (in this case an HTTPs URL).
+The command above creates the import `.dvc` file `data.xml.dvc`, that contains
+an external dependency (in this case an HTTPs URL).
 
 <details>
 
-### Expand to see resulting DVC-file
+### Expand to see resulting `.dvc` file
```

Comment:
This is causing a bug in the engine:

![image](https://user-images.githubusercontent.com/1477535/91612796-c5de9200-e943-11ea-8ecc-c6c83ec5ab5f.png)

> From https://dvc-landing-jorge-1wquguk7rhkq.herokuapp.com/doc/user-guide/external-dependencies#example-import-url-command

(Same in line 196)

Somewhat related to #1614 I think... Will comment there.

Reply:
For now I'll remove the back quotes in these changes.

Reply:
This actually just seems to be an issue with the `details` component. Let me demonstrate with the source code:

This is how the `details` component grabs text from its children:

```
const filteredChildren = children.filter(child => child !== '\n')

if (!filteredChildren.length) return null
if (typeof filteredChildren[0] === 'string') 

const text = filteredChildren[0].props.children[0]
```

This is a cleaned up version of what `filteredChildren[0]` looks like with inline code:

```
{
  type: 'h3',
  props: {
    id: 'expand-to-see-resulting-dvc-file-1',
    style: { position: 'relative' },
    children: [
      'Expand to see resulting ',
      {
        type: 'code',
        props: { children: [ '.dvc' ] },
      },
      ' file',
      {
        type: [Function: Link],
        props: {
          children: [
            {
              type: 'svg',
              props: {
              },
            }
          ]
        },
      }
    ]
  }
}
```

I think I can fix this by using the `h3`'s children literally instead of just pulling the first child's text.

Reply:
Fixed the issue at #1736, if you'd like to keep the code quotes in the trigger. Here's a screenshot of what it looks like:

![fixed-details](https://user-images.githubusercontent.com/9111807/91623323-03a1e180-e969-11ea-9cd4-e5df7df6cba9.png)

Reply:
🙏 thank you @rogermparent !